flake.nix

```
@@ -3,17 +3,17 @@
   inputs = {
     cardano-node.url = "github:intersectmbo/cardano-node";
     cardano-configurations = {
-      url = "github:input-output-hk/cardano-configurations?rev=fa19a63c9a2050940583b10e9f39ed02e3cf306f";
+      url = "github:input-output-hk/cardano-configurations?rev=6d7fac44d6a877ae9a3cf5eeda163ae539b17774";
```

```suggestion
      url = "github:input-output-hk/cardano-configurations/6d7fac44d6a877ae9a3cf5eeda163ae539b17774";
```

You can rewrite it like this if you prefer, nothing should change I believe.